townhall/myapi/tests/test_volunteer.py

```diff
@@ -1,73 +1,108 @@
 from django.test import TestCase
+from django.core.management import call_command
+from unittest.mock import patch
 from myapi import models as townhall_models
-from myapi import services as townhall_services
 
 # VOLUNTEER
 
 # Testing ALL tests "python manage.py test myapi.tests"
 # Testing ONLY volunteer "python manage.py test myapi.tests.test_volunteer"
 
+@patch('myapi.services.VolunteerServices')
```

Comment:
Excellent use of patch decorator, patch also allows the mocking of methods so like `@patch('myapi.services.VolunteerServices.get_volunteer')`